README.md

```diff
@@ -1,28 +1,28 @@
-# LaTeX-proposal:  A set of LaTeX classes for preparing proposals
+# LaTeX-proposal: A set of LaTeX classes for preparing proposals
 
 Writing grant proposals is a collaborative effort that requires the integration of
 contributions from many individuals. The use of an ASCII-based format like LATEX allows to
 coordinate the process via a source code control system like Git or Subversion, allowing
 the proposal writing team to concentrate on the contents rather than the mechanics of
 wrangling with text fragments and revisions. In fact the proposal package has evolved out
 of a series of collaborative proposal writing efforts, where large teams (up to 30
-individuals from up to 20 sites) have written a 100- page proposal in three weeks (with
+individuals from up to 20 sites) have written a 100-page proposal in three weeks (with
 over 2000 commits). Such collaborative writing sprints are impossible without a revision
 control system and a “semantic” document class that generates tables, charts, and
 deliverable lists from content markup and thus takes care of many of the routine tasks of
 keeping information consistent.
 
 ## Using the LaTeX Proposal Class
 
-In the simplest case, just clone the repository, and extend your  `TEXINPUTS`
+In the simplest case, just clone the repository, and extend your `TEXINPUTS`
 environment variable so that it can find it. On a UNIX system something like the following
-should work. 
+should work.
 ```
-cd /path/to/your/project
+cd /path/to/project
 git clone https://github.com/KWARC/LaTeX-proposal
-echo 'export TEXINPUTS = "$(TEXINPUTS):/path/to/your/project/LaTeX-proposal//:"' >> ~/.bashrc
```

Comment:
actually, we need the // in the path, since the sytle filess are in subdirectories.